pkgs/development/python-modules/pysnmp-pyasn1/default.nix

```diff
@@ -35,6 +35,7 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "Python ASN.1 encoder and decoder";
     homepage = "https://github.com/pysnmp/pyasn1";
+    changelog = "https://github.com/pysnmp/pyasn1/releases/tag/v${version}";
```

Comment:
```suggestion
```
It's useless cause it isn't kept up to date.

Reply:
Look up-to-date for me: https://github.com/pysnmp/pyasn1/releases/tag/v1.1.3

[CHANGES.rst](https://github.com/pysnmp/pyasn1/blob/main/CHANGES.rst) is no longer maintained.

Reply:
Oops, I should have read more carefully.